sdks/java/extensions/sorter/build.gradle

```diff
@@ -36,10 +36,10 @@ hadoopVersions.each {kv -> configurations.create("hadoopVersion$kv.key")}
 dependencies {
   compile project(path: ":sdks:java:core", configuration: "shadow")
   compile library.java.vendored_guava_26_0_jre
+  compile library.java.jsr305
+  compile "org.slf4j:slf4j-api:1.7.30"
```

Comment:
This should be `library.slf4j_api` and please set up the version in `BeamModulePlugin.groovy`. The same for all libraries.